docs/building_models.md

```diff
@@ -137,11 +137,18 @@ run recovery ID without the final underscore and digit.
 
 ### Testing an existing model
 
+##### From a run_recovery:
 As for continuing training, but set `--train` to `False`. Thus your command should look like this:
+
```

Comment:
Can you add a bit more plaintext here? something like "to evaluate an existing model on a test test, this model can come from an previous training run in AzureML, or a local checkpoint."